catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -844,15 +848,11 @@ private SolrQuery logicalOperator(List<SolrQuery> operands, String operator) {
                 throw new UnsupportedOperationException("Null operand found");
             }
             String operandAsString = operand.toString();
-            try {
-                if (operandAsString.contains(URLEncoder.encode(SCORE_DISTANCE, "UTF-8"))) {
-                    SolrQuery temp = operands.get(0);
-                    operands.set(0, operand);
-                    operands.set(i, temp);
-                    break;
-                }
-            } catch (UnsupportedEncodingException e) {
-                LOGGER.info("Unable to encode {}", SCORE_DISTANCE, e);
+            if (operandAsString.contains(partialUrlEncode(SCORE_DISTANCE))) {
```

Comment:
That sounds better to me as well. I was wondering why we had to pull in that `partialUrlEncode` method.